apps/shipments/signals.py

```diff
@@ -117,6 +118,15 @@ def trackingdata_post_save(sender, **kwargs):
                                             {"type": "tracking_data.save", "tracking_data_id": instance.id})
 
 
+@receiver(post_save_changed, sender=Shipment, fields=['delivery_act'], dispatch_uid='shipment_delivery_act_post_save')
+def shipment_delivery_act_changed(sender, instance, changed_fields, **kwargs):
+    if instance.delivery_act <= datetime.now():
```

Comment:
If we want to validate the value of delivery_act, we should do this in the ShipmentUpdateSerializer, not here. Either move this check there or remove it.